python/Ganga/Runtime/bootstrap.py

```diff
@@ -181,6 +181,10 @@ def manualExportToGPI(my_interface=None):
     from Ganga.GPIDev.Lib.Registry.JobRegistry import jobSlice
     exportToInterface(my_interface, "jobSlice", jobSlice, "Functions")
 
+    """Needed for the JobTime if we plan to use the interface to eval the XML as well"""
+    import datetime
+    exportToInterface(my_interface, "datetime", datetime, 'Classes')
```

Comment:
I am very much against adding more code (which is more complexity and maintenance effort) in a branch which is supposed to be improving performance. More code will almost always make things slower and constantly patching small problems across the code base without really understanding what's happening at the deeper levels is what has been causing problems for a long time. I guarantee that we could get equal performance improvements by removing and simplifying code rather than adding more for special cases etc.